tests/suite/layout/inline/text.typ

```diff
@@ -70,7 +70,7 @@ I saw Zoe yӛsterday, on the tram.
 
 --- text-tracking-changed-temporarily ---
 // Test tracking for only part of paragraph.
-I'm in#text(tracking: 0.15em + 1.5pt)[ spaace]!
+I'm in#text(tracking: 0.15em + 1.5pt)[ space]!
```

Comment:
there didn't seem to be any particular reason to double the `a` here. And this repository didn't appear to be particularly playful.

Reply:
It's a reference to this: https://www.youtube.com/watch?v=xeKMS62GrTI

The extra a gives it a nicer touch imo ^^

Reply:
Ah, I'm not a portal person (otherwise I'd have gotten the reference). The `playful` bit in my highlight is my note that it seemed _possible_ that it was a reference to something :).

It looks like they had `spaaace`. I'm happy to drop this. I would have been less likely to change it if there were more 'a's.